### PR TITLE
Removed tracking 'comments section viewable' ahoy event

### PR DESCRIPTION
**Pull request by lightalloy**
## What type of PR is this? (check all applicable)
- [x] Refactor

## Description
Removed "Comment section viewable" ahoy event, discussed in https://github.com/forem/forem-internal-eng/issues/523#issuecomment-1506086754

## Related Tickets & Documents
- Related Issue https://github.com/forem/forem-internal-eng/issues/523

## QA Instructions, Screenshots, Recordings
The comments section should be viewable/usable as before.

## Added/updated tests?

- [ ] Yes
- [x] No, and this is why: haven't added any logic
- [ ] I need help with writing tests

### CODE REVIEW

**Review by jaw6 — APPROVED** [review]

**Comment by lightalloy** [comment]
I think I have removed focusing on the textarea along with the event :) , I'll fix that.

**Comment by lightalloy** [comment]
@jaw6 @lboogie04 please, re-review, I have returned focusing on the comment textarea when there is a `#comments` anchor (a cypress test related to that was failing).

File: app/javascript/packs/articlePage.jsx [hunk 1]
```diff
@@ -88,26 +87,13 @@ function showAnnouncer() {
...
-  const callback = (entries, observer) => {
-    entries.forEach((entry) => {
-      if (entry.isIntersecting) {
-        ahoy.track('Comment section viewable', { page: location.href });
-        observer.disconnect();
-      }
-      if (location.hash === '#comments') {
-        //handle focus event on text area
-        const element = document.getElementById('text-area');
-        const event = new FocusEvent('focus');
```
Comment (jaw6) [inline app/javascript/packs/articlePage.jsx:102]: Ah, I see it now :+1:

**Review by jaw6 — APPROVED** [review]

File: app/javascript/packs/articlePage.jsx (hunk 1 above)
Comment (lightalloy) [reply app/javascript/packs/articlePage.jsx:102]: The method was doing 2 things (tracking the event + focusing), so it was harder to spot.

**Pull request merged**